engine/src/main/java/org/hibernate/search/bridge/builtin/time/impl/DurationNumericBridge.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.bridge.builtin.time.impl;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+
+/**
+ * Converts an {@link Duration} to the number of milliseconds from the epoch of 1970-01-01T00:00:00Z.
```

Comment:
A duration is not related to the epoch, right? Rather it's stored as the duration's length in ms.